sentry_sdk/hub.py

```diff
@@ -161,6 +161,12 @@ def add_breadcrumb(self, *args, **kwargs):
             logger.info("Dropped breadcrumb because no client bound")
             return
 
+        hint = kwargs.pop("hint", None)
+        if not hint:
+            hint = {}
+        else:
+            hint = dict(hint)
```

Comment:
Do you need to catch errors here for when the argument cannot be converted into a dict?